gui/src/components/Restores.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+    <div>
+        <h4 class="py-3">Restore Statuses in Cluster: {{cluster.cluster_name}}</h4>
+        <ClusterSummary :cluster="cluster"/>
+        <div class="row justify-content-end mb-3 pr-3 py-3">
+            <button
+                    type="button"
+                    class="btn btn-primary mx-1"
+                    @click="viewBackups()"
+            >View Backups
+            </button>
+        </div>
+        <table class="table table-bordered text-center">
```

Comment:
Same as the backup table, if there are no backups, should there be a text or a visual cue displayed below the header that says something like
"There are no restores yet"